Extending/Packages/UmbPack/index.md

```diff
@@ -10,11 +11,17 @@ meta.Description: "How to use the UmbPack tool to deploy package versions to Our
 UmbPack is an open source NuGet tool that can deploy packages to Our, and also help you pack your files into a package.
 
 :::note
-You still need to create the package on Our before the tool can be used. Once it has been set up you will then be able to add new versions of the package zip files using UmbPack
+**Are you using Umbraco 9+?**
+
+As of Umbraco 9 all packages needs to be installed through NuGet, and it is no longer to install packages from .zip files.
```

Comment:
Should this also reference the blogpost (https://umbraco.com/blog/packages-in-umbraco-9-via-nuget/) explaining this change?

```suggestion
As of Umbraco 9, packages can only be installed using NuGet and it is no longer possible to install packages from package zip files.
```